dashboard/src/actions/apps.test.tsx

```diff
@@ -65,3 +65,34 @@ describe("fetches applications", () => {
     });
   });
 });
+
+describe("delete applications", () => {
+  const deleteApp = App.delete;
```

Comment:
as suggested in https://github.com/kubeapps/kubeapps/pull/563#discussion_r214469388, I think this could be called `deleteAppOrig` or something